thirdparty/nginx/nginx-1.7.1-zimbra/src/core/ngx_zm_lookup.c

```diff
@@ -2131,8 +2142,9 @@ ngx_zm_lookup_get_http_alias_key (ngx_pool_t *pool, ngx_log_t *log,
 }
 
 static ngx_str_t
-ngx_zm_lookup_get_id_route_key (ngx_pool_t *pool, ngx_log_t *log,
-    ngx_str_t proto, ngx_str_t id, ngx_flag_t isAdmin)
+ngx_zm_lookup_get_id_route_key(ngx_pool_t *pool, ngx_log_t *log,
+        ngx_str_t proto, ngx_str_t id, ngx_flag_t isAdmin,
```

Comment:
Is it better to program this function with single parameter that decides on admin or zx.

Reply:
Passing single variable like work will reduce the extra parameter, without much change in caller.